packages/web3-providers/package.json

```diff
@@ -5,12 +5,14 @@
   "types": "./dist/index.d.ts",
   "dependencies": {
     "@masknet/plugin-infra": "workspace:*",
-    "@masknet/web3-shared-evm": "workspace:*",
     "@masknet/web3-shared-base": "workspace:*",
-    "bignumber.js": "^9.0.1",
+    "@masknet/web3-shared-evm": "workspace:*",
+    "bignumber.js": "9.0.1",
     "date-fns": "^2.27.0",
     "json-stable-stringify": "^1.0.1",
     "lodash-unified": "1.0.1",
+    "socket.io-client": "2.4.0",
```

Comment:
Can you use newer version?

Reply:
The socket version is specified when use zerion: https://docs.zerion.io/websockets/getting-started/installation